src/main/java/com/artipie/SliceFromConfig.java

```diff
@@ -101,6 +102,14 @@ public SliceFromConfig(final RepoConfig config, final FileSystem fs) {
                             );
                         }
                     ),
+                    new MapEntry<>(
```

Comment:
@olegmoz  I would also extract `php` creation part to a private method to make code more readable